jax/_src/api.py

```diff
@@ -212,6 +212,7 @@ def jit(
   device: Optional[xc.Device] = None,
   backend: Optional[str] = None,
   donate_argnums: Union[int, Iterable[int]] = (),
+  inline: bool = False,
 ) -> F:
   """Sets up ``fun`` for just-in-time compilation with XLA.
```

Comment:
I'd document inline.

Reply:
Do you like it as a public API? (Seems pretty reasonable to me! But I wanted to double check.)

Reply:
I think so.

The only caveat is at some point in the future we might also want users to provide a hint to, say, XLA on whether it should inline functions. Perhaps that could be solved by `inline="xla"` or something at that time.

Reply:
For that case I propose an `inline2` option.